modules/network/smb2man/libsmb2/lib/socket.c

```
@@ -0,0 +1,1510 @@
+
+int
+smb2_write_to_socket(struct smb2_context *smb2)
+{
+        struct smb2_pdu *pdu;
+
+        if (!SMB2_VALID_SOCKET(smb2->fd)) {
+                smb2_set_error(smb2, "trying to write but not connected");
+                return -1;
+        }
+        while ((pdu = smb2->outqueue) != NULL) {
+                struct iovec iov[SMB2_MAX_VECTORS] _U_;
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If _U_ is a macro then please configure it.

(unknownMacro)